images/postgres-operator/metadata.yaml

```diff
@@ -0,0 +1,10 @@
+name: postgres-operator
+image: cgr.dev/chainguard/postgres-operator
+logo: https://storage.googleapis.com/chainguard-academy/logos/postgres-operator.svg
+endoflife: ""
+console_summary: ""
+short_description: TODO
+compatibility_notes: ""
+readme_file: README.md
+upstream_url: https://github.com/wolfi-dev/os
+keywords: []
```

Comment:
Can we add some keywords? probably things like 'application', but can have a look through our other image examples

Reply:
thanks, added